src/Entity/ProductData.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use App\Repository\ProductDataRepository;
+use DateTime;
+use DateTimeInterface;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+#[ORM\Table(name: 'tblProductData')]
+#[ORM\Entity(repositoryClass: ProductDataRepository::class)]
+#[UniqueEntity(fields: ['strProductCode'])]
+class ProductData
+{
+    #[ORM\Id]
+    #[ORM\Column(name: 'intProductDataId', type: 'integer', nullable: false, options: ['unsigned' => true])]
+    #[ORM\GeneratedValue(strategy: 'IDENTITY')]
+    private int $intproductdataid;
```

Comment:
it's not a good practice to name entity variables like "int... str..." could you just rename it to "productDataId" and so on?